snakes_cafe.py

```diff
@@ -160,7 +149,7 @@ def __init__(self):
         self.subtotal = 0
```

Comment:
Just so you know, putting the current state of the order as a property of this class instead of within the menu would be more fitting with object-oriented programming.